tests/functional/DeployHookTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Unish;
+
+use Webmozart\PathUtil\Path;
+
+/**
+ *  @group slow
+ *  @group commands
+ */
+class DeployHookTest extends CommandUnishTestCase
```

Comment:
Thanks for writing a test! We prefer to extend UnishIntegrationTestCase when possible. Those tests are faster. If you run into problems, then this is fine.

Reply:
All tests that use the woot module use CommandUnishTestCase, I could not easily transform the test to UnishIntegrationTestCase.

Reply:
There is one such Integration case - \Unish\AnnotatedCommandCase. Lets explore using Integration in a future follow-up.